analytics/upress_reporting/helpers/download_data_aggregator.py

```diff
@@ -114,6 +123,10 @@ def _match_log_info_with_frbr_data(self, log_object):
             title_parse = ""
             id_parse = None
 
+            '''
```

Comment:
Thoughts on having this method return an InteractionEvent rather than a tuple? That may have been your original idea but just wanted to reiterate :)

Reply:
yes that sounds great!